dashboard/app/models/script.rb

```diff
@@ -1680,6 +1670,16 @@ def summarize_i18n_for_display(include_lessons=true)
     data
   end
 
+  # Returns summary object of all the course versions that an instructor can
+  # assign or all the launched versions a participant can view
```

Comment:
I don't think I follow -- how do you know which of these two things to return?

Reply:
Update the comment to explain it in more detail

Reply:
OK, I see what you are saying / doing. it's a bit of an antipattern to have these two things crammed into the same method, which I'm assuming stems from the fact that there is just one summarize method trying to do two different things. I think this is ok for now, but I would suggest that a more ideal arrangement would be to have more specific summarize methods for different purposes, like we do in other models. 

this is not blocking on this PR or this deadline -- I know you've got a lot to do before launch. but if you need to touch the summarize method again, maybe consider splitting it.

Reply:
Makes sense. It was mirroring the way it was set up before which is why I went with this set up.